openedx/openedx-platform
# feat: new URL to render a v2 xblock in an iframe [FC-0062]

## Description
This PR adds an endpoint in the XBlock rest API to allow rendering it in the course authoring mfe

## More information
Part of:
 - https://github.com/openedx/frontend-app-course-authoring/issues/1189

## Testing Instructions
Check https://github.com/openedx/frontend-app-course-authoring/pull/1242 
___
Private ref: [FAL-3801](https://tasks.opencraft.com/browse/FAL-3801)
